dashboard/src/app/index.module.js

```diff
@@ -96,8 +96,51 @@ initModule.factory('AuthInterceptor', ($window, $cookies, $q, $location, $log) =
   };
 });
 
-initModule.config(['$routeProvider', '$locationProvider', '$httpProvider', ($routeProvider, $locationProvider, $httpProvider) => {
+initModule.factory('AddMachineTokenToUrlInterceptor', ($injector, $q) => {
```

Comment:
this is not how factory are registered
it is defined in a dedicated class and plugged through register mechanism in a config object
Please fix